### PR TITLE
ConfigurationResolver - grooming

### PR DESCRIPTION
**Pull request by SpacePossum**
- resolve differ in configuration resolver, not fin fix command
- configuration resolver return `null` for cache file if caching is disabled
- add test to test command line args. override default config
- add test to test config file overrides default config
- typo in readme
- configuration resolver; sort method names

### CODE REVIEW

File: src/Console/Command/FixCommand.php [hunk 1]
```diff
@@ -87,7 +87,7 @@ public function __construct()
     }
 
     /**
-     * {@inheritdoc}
+     * @see Command
```
Comment (keradus) [inline src/Console/Command/FixCommand.php]: please revert

**Comment by keradus** [comment]
:+1: if green

**Comment by SpacePossum** [comment]
Travis timeout on 7, other versions all green (including 7.1)

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**